pkg/cmd/roachtest/bank.go

```diff
@@ -327,6 +327,29 @@ func (s *bankState) startSplitMonkey(ctx context.Context, d time.Duration, c *cl
 	}()
 }
 
+// IsExpectedRelocateError maintains an allowlist of errors related to
```

Comment:
Where did these diffs come from? Do you mind adding references to them in the commit message?

Reply:
Works for me!